cmd/acmesolver/main.go

```diff
@@ -27,15 +29,16 @@ import (
 // cert-manager.
 
 func main() {
-	stopCh, exit := util.SetupExitHandler(util.GracefulShutdown)
+	ctx, exit := util.SetupExitHandler(context.Background(), util.GracefulShutdown)
 	defer exit() // This function might call os.Exit, so defer last
 
 	logf.InitLogs()
 	defer logf.FlushLogs()
+	ctx = logf.NewContext(ctx, logf.Log, "acmesolver")
```

Comment:
Oh, I see that you've moved it here. But why?

Reply:
I moved the New context logic to the main.go file so we also have a context with logger for our ExecuteContext call.